app/models/manageiq/providers/base_manager/event_catcher.rb

```diff
@@ -19,4 +19,8 @@ def friendly_name
   def self.kill_priority
     MiqWorkerType::KILL_PRIORITY_EVENT_CATCHERS
   end
+
+  def self.service_base_name
+    "manageiq-providers-#{minimal_class_name.underscore.tr("/", "_")}"
```

Comment:
can we do a super class here?

Reply:
Yeah I considered that but some of these are queue workers and some are not so they have different parent classes.  Also considered a `ProviderWorkerMixin` but this would be the only thing we could share.  Maybe better to update `minimal_class_name` to include the `providers-` prefix?

Reply:
ProviderWorkerMixin came in handy here